string serial_force_state
int32 force